docs/Examples/Windows/web.config

```diff
@@ -0,0 +1,64 @@
+<configuration>

```

Comment:
I added the web.config information directly into the doc. Do you want me to fork it again and just leave off the 'Windows' folder? just dropping IIS into the 'Examples' folder?